orbs/changelog/Dockerfile.publish

```diff
@@ -13,10 +15,21 @@ RUN apt-get update -y && \
       # checkout
       git openssh-client \
       # node installation
-      bash curl \
+      bash curl wget \
     && \
-    curl -sL https://deb.nodesource.com/setup_10.x | bash - && \
-    apt-get install -y --no-install-recommends nodejs && \
+    ( \
+      cd /tmp && \
+      wget -q https://nodejs.org/download/release/v$NODE_VERSION/node-v$NODE_VERSION-linux-x64.tar.gz && \
+      tar -xzf node-v$NODE_VERSION-linux-x64.tar.gz && \
+      rm node-v$NODE_VERSION-linux-x64.tar.gz && \
+      cd node-v$NODE_VERSION-linux-x64 && \
+      cp -r bin /usr/local/ && \
+      cp -r include /usr/local/ && \
+      cp -r lib /usr/local/ && \
+      cp -r share /usr/local/ && \
+      rm -rf node-v$NODE_VERSION-linux-x64 \
```

Comment:
__rmk__

Instead of manually download, copy file and remove, perhaps it would be shorter to install `NVM` and install node through nvm ?

Or use instruction in https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions. It may not be precisely `14.17.5`, but as it is a LTS, updates will only be security ones, so good to take

Reply:
Installing nvm may be shorter but it's not strictly necessary to install node. It may seem like a lot of steps but note that it's how the current `jobteaser/node` image is done.
Also, it's not something that would require an update in the future, only the `NODE_VERSION` & `NPM_VERSION` would be updated.

The approach using https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions is the one that was used before but as you mentioned it's not precise enough to let us choose exactly which version of node we want. We changed it to use what you see here in order to be able to provide an exact version of Node and NPM.
I guess wether it's that important or not to control the Node & NPM version so precisely is debatable, but it would allow us to enforce developers to also use those same exact versions locally.

Open to debate 🙂 ...

Reply:
@happypoulp it's not a strong opinion, but i would say that it's better to stay up to date on v14. Only security updates could happen as it is on LTS, and it could be better to always build on more secure platform.

NPM is a bit different, Fabien for example is a bit "frilleux" to not strictly fix version